combat/pycombat.py

```diff
@@ -343,15 +346,15 @@ def treat_batches(batch):
     Returns:
         n_batch {int} -- number of batches
         batches {int list} -- list of unique batches
-        n_batches {int list} -- list of batches lengths 
+        n_batches {int list} -- list of batches lengths
         n_array {int} -- total size of dataset
     """
     n_batch = len(np.unique(batch))  # number of batches
     print("Found "+str(n_batch)+" batches.")
     batches = []  # list of lists, contains the list of position for each batch
     for i in range(n_batch):
         batches.append(np.where(batch == np.unique(batch)[i])[0])
-    batches = np.asarray(batches)
+    batches = np.asarray(batches, dtype=object)
```

Comment:
Just to confirm -- what would be the purpose of this? As far as I know dtype object allows strings to be included in the array, which is not exactly what we aim for here.

Reply:
I added this to avoid depreciation warning from numpy, but you are right dtype should be more specific. I guess **int32** type should work for batch indices in this case.

Reply:
Great! Thanks for your (quick) answer!